styles/main.css

```diff
@@ -1,5 +1,13 @@
-body {
+html {
   box-sizing: border-box;
+}
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
   background-color: #bdfe00;
```

Comment:
box-sizing is not inherited by default, so setting on just body doesn't apply it to all elements